tests/integration/commands/env/env-list.test.ts

```diff
@@ -0,0 +1,50 @@
+import { describe, expect, test } from 'vitest'
+
+import { FixtureTestContext, setupFixtureTests } from '../../utils/fixture.js'
+
+import routes from './api-routes.js'
+
+describe('env:list command', () => {
+  setupFixtureTests('empty-project', { mockApi: { routes } }, () => {
+    test.concurrent<FixtureTestContext>('should return the object of keys and values', async ({ fixture }) => {
```

Comment:
you could set the test context for the whole vitest module:

```typescript
declare module 'vitest' {
  export interface FixtureTestContext {
    fixture:... 
  }
}
```
then you don't have to add the generic on each test